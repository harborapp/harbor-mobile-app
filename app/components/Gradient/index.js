import React from 'react'
import LinearGradient from 'react-native-linear-gradient'

const Gradient = (props) => {
  return (
    <LinearGradient colors={props.colors}>
      {props.children}
    </LinearGradient>
  )
}

export default Gradient
