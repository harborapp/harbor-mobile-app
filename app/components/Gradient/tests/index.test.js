/* eslint-env jest */
import React from 'react'
import renderer from 'react-test-renderer'

import Gradient from '../'

describe('Gradient', () => {
  it('should render', () => {
    const tree = renderer.create(
      <Gradient colors={['red', 'green']}>
        <React.Fragment />
      </Gradient>
    )

    expect(tree).toMatchSnapshot()
  })
})
