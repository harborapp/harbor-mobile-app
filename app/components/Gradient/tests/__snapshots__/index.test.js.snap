// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gradient should render 1`] = `
<BVLinearGradient
  colors={
    Array [
      4294901760,
      4278222848,
    ]
  }
  endPoint={undefined}
  locations={null}
  startPoint={undefined}
/>
`;
